### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by AtheerAlgherairy** [ConvLab/ConvLab-3#99]
Trying to use **DST** for **sgd** dataset but I got the following error:

![Screen Shot 1444-05-13 at 12 25 58 PM](https://user-images.githubusercontent.com/7421065/206140568-8fe8d720-a5be-4e24-9bca-66f6df9ea1a4.png)

![Screen Shot 1444-05-13 at 12 26 12 PM](https://user-images.githubusercontent.com/7421065/206140609-c10e77a0-fcbc-425c-86c0-97fa29a76b79.png)

I am using (update) method in DST class similar to convlab-2..

**Comment by zqwerty** [comment]
BTW, please give a title for the issue to facilitate other users that may have similar question

**Comment by zqwerty** [comment]
You could also refer to https://github.com/ConvLab/ConvLab-3/blob/d19e48a30abca679aef42865146c2d7bccce8369/convlab/dialog_agent/agent.py#L45

to find how to initialize and push new utterances into the state of the `DST` class.

**Comment by AtheerAlgherairy** [comment]
**Thank you.. It works like this:**

![Screen Shot 1444-05-13 at 1 30 33 PM](https://user-images.githubusercontent.com/7421065/206155704-70bf487d-3d6b-470f-89aa-1ca9a8140406.png)

So, this **T5DST** model doesn't need the output from **nlu**?

**Comment by zqwerty** [comment]
Yes, it predicts state from utterances directly. however, the RL policy module need user dialog acts (nlu output)
